hls4ml/converters/keras/convolution.py

```diff
@@ -85,4 +85,4 @@ def parse_conv2d_layer(keras_layer, input_names, input_shapes, data_reader, conf
     else:
         output_shape = [input_shapes[0][0], layer['out_height'], layer['out_width'], layer['n_filt']]
 
-    return layer, output_shape
+    return layer, output_shape
```

Comment:
Now there is a difference here. Again, it would be good if this file could be unchanged wrt master branch